docs/getting-started-guides/scratch.md

```diff
@@ -258,7 +258,7 @@ You will end up with the following files (we will use these variables later on)
 The admin user (and any users) need:
 
   - a token or a password to identify them.
-  - tokens are just long alphanumeric strings, e.g. 32 chars.  See
+  - tokens are just long alphanumeric strings, for example, 32 chars.  See
```

Comment:
This might be easier to rephrase entirely, moving the "for example" to the end:

    tokens are just long alphanumeric strings, 32 chars for example.